docs/edgecases.md

```diff
@@ -52,99 +69,118 @@ building
-  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
-  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
-  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
-  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
-  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
-  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
-  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
-  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
-  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
-  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
-  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
-ModuleNotFoundError: No module named 'poetry'
-
-Removed file:///build/source from build tracker '/build/pip-req-tracker-6nbzvq94'
-Removed build tracker: '/build/pip-req-tracker-6nbzvq94'
+  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
+  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
+  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
+  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
+  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
+  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
+  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
+ModuleNotFoundError: No module named 'setuptools'
+
+error (ignored): error: cannot unlink '/tmp/nix-build-python3.10-django-4.1.3.drv-1/Django-4.1.3': Directory not empty
+error: 1 dependencies of derivation '/nix/store/hz4b87s99s1lwiz2m0vwilhlh6rlfj64-python3-3.10.8-env.drv' failed to build
+error: 1 dependencies of derivation '/nix/store/60gaxbdf11lhaj9xg50cf0rr6x5v8v1z-nix-shell-env.drv' failed to build
 ```
+
+As you can see on the fourth last line it's missing `setupools` which in this case is our missing `PACKAGENAME`.
+
 </details>
 
-In order to be able to build `extralib` we should modify our nix definition as follows:
+In order to be able to build `django-floppyforms` we should modify our nix definition as follows:
+
+``` nix
+poetry2nix.mkPoetryApplication {
+  projectDir = ./.;
+  overrides = poetry2nix.defaultPoetryOverrides.extend
+    (self: super: {
+      django-floppyforms = super.django-floppyforms.overridePythonAttrs
+      (
+        old: {
+          buildInputs = (old.buildInputs or [ ]) ++ [ super.buildtools ];
+        }
+      );
+    });
+}
 ```
+
+This override will instruct the underlying build logic to include the additional build dependency into the inputs of `django-floppyforms`.
+It might help to know that you don't need to use the full package name like `python39Packages.setupools` but can just use `setuptools` directly.
+However you have to use the name like defined in `Nixpkgs` so something like `flit_scm` becomes `flit-scm`.
```

Comment:
Not correct, the name should be normalized as per PEP-517. Also, nixpkgs isn't a literal piece of code so shouldn't be in `.`